pip/qsharp/openqasm/__init__.py

```diff
@@ -4,15 +4,15 @@
 from ._circuit import circuit
 from ._compile import compile
 from ._estimate import estimate
-from ._import import import_qasm
+from ._import import import_openqasm
 from ._run import run
 from .._native import ProgramType, OutputSemantics, QasmError  # type: ignore
 
 __all__ = [
     "circuit",
     "compile",
     "estimate",
-    "import_qasm",
+    "import_openqasm",
```

Comment:
It's too bad this can't just be called `import` but I'm guessing that's because that is already a Python keyword, right?

Reply:
Yup, that was my first choice.